File: my-messenger-backend/my.messenger.backend/WS/StreamMessage.cs
using my.messenger.common.Messaging;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace my.messenger.backend.WS
{
    public class StreamMessage
    {
        public StreamMessageType StreamMessageType { get; set; }

        public Message Message { get; set; }

        public String TokenId { get; set; }
    }
}

File: my-messenger-backend-java/src/main/java/mymessenger/backend/ws/StreamMessage.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package mymessenger.backend.ws;

import mymessenger.backend.model.messaging.Message;

/**
 *
 * @author guilherme
 */
public class StreamMessage {
    
    private StreamMessageType streamMessageType;
    
    private Message message;
    
    private String tokenId;

    public StreamMessageType getStreamMessageType() {
        return streamMessageType;
    }

    public void setStreamMessageType(StreamMessageType streamMessageType) {
        this.streamMessageType = streamMessageType;
    }

    public Message getMessage() {
        return message;
    }

    public void setMessage(Message message) {
        this.message = message;
    }

    public String getTokenId() {
        return tokenId;
    }

    public void setTokenId(String tokenId) {
        this.tokenId = tokenId;
    }
    
    
}
